### DISCUSSION TITLE
Saving successives solutions

### DISCUSSION THREAD

**Discussion by htavv** [coin-or/Cbc#403] [Q&A]
Hello,

First of all, thanks for the work you are doing on Cbc !

I use Cbc 2.10.5 command line to solve MIPs and I would like to get successive solutions as soon as they are found, without stopping the solving process.

Is there a way to save these successive solutions in files ?

Thank you.

Thomas

**Comment by jhmgoossens** [comment]
I believe (but could be wrong) that this can be done using the `CbcEventHandler` class. See for example [cbc\examples\inc.cpp](https://github.com/coin-or/Cbc/blob/master/examples/inc.cpp#L67) line 67.

**Reply by htavv** [reply]
Thank you for your reply.
However I use the command line, not the C++ interface directly, and I was wondering if it is possible to do it directly via an option.
